### PR TITLE
add option to skip create for perf tests

### PR DESCRIPTION
**Pull request by rkpagadala**
Separate creation of vm/cluster and and installing and setting them up for testing.

### CODE REVIEW

File: tools/setup_perf_cluster.sh [hunk 1]
```diff
@@ -71,7 +89,6 @@ function ExecuteEval() {
 
 function create_cluster() {
   Execute gcloud container clusters create $CLUSTER_NAME $GCP_OPTS --machine-type=$MACHINE_TYPE --num-nodes=$NUM_NODES --no-enable-legacy-authorization
-  Execute gcloud container clusters get-credentials $CLUSTER_NAME $GCP_OPTS
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: why is this gone?

File: tools/setup_perf_cluster.sh [hunk 2]
```diff
@@ -473,6 +494,9 @@ function check_image_versions() {
 if [[ $SOURCED == 0 ]]; then
   # Normal mode: all at once:
   update_gcp_opts
+  if [ "${SKIP_CREATE}" != "true" ]; then
+    create_vm_cluster_all
+  fi
   setup_all
```
Comment (ldemailly) [inline tools/setup_perf_cluster.sh]: I haven't used the script as is in a long time, I only "source" it and run functions 1 by 1

I think "run_tests" probably doesn't work the first time as it takes a while to get the ips for ingress for a brand new cluster, so in theory we would need a "wait_for_ready" before "run_tests" below

I think also in automated way, you probably don't need to start "fortio report &"

Would you consider sourcing the file and calling the functions individually maybe ?

File: tools/setup_perf_cluster.sh (hunk 1 above)
Comment (rkpagadala) [reply tools/setup_perf_cluster.sh]: there is another function kubectl_setup that does it. This one is redundant.

File: tools/setup_perf_cluster.sh (hunk 2 above)
Comment (rkpagadala) [reply tools/setup_perf_cluster.sh]: It will be another wrapper script, I can give it a go

**Comment by rkpagadala** [comment]
Adding a wrapper script, this PR is not needed

**Pull request closed without merging**